### PR TITLE
Complete Trace processing section Panels (section Trace processing completed)

### PR DESCRIPTION
**Pull request by mca-sh**
Pages include:
* Project management
* Sample management
* Plot
* Sub-images
* Background correction
* Factor corrections
* Denoising
* Photobleaching
* Find states
* Visualization area

### CODE REVIEW

File: docs/assets/images/equations/equations.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Latex expressions of equations, images generated by https://www.codecogs.com/latex/eqneditor.php
```
Comment (pflanzenhueter) [inline docs/assets/images/equations/equations.txt]: I need to understand this. There do you have this notation from?

File: docs/histogram-analysis/workflow.md [hunk 2]
```diff
@@ -115,6 +115,7 @@ The relative population
 It can be estimated from the histogram by integrating each peak to a value 
 [*S*<sub>*j*</sub>](){: .math_var } and calculating the ratio:
 
+{: .equation }
 <img src="../assets/images/equations/HA-eq-pop.gif" alt="X_{j} = \frac{S_{j}}{\sum_{j'=1}^{J}( S_{j'})}">
```
Comment (pflanzenhueter) [inline docs/histogram-analysis/workflow.md]: Ok. I need to understand, why equations are implemented as gif files.

File: docs/trace-processing/panels/panel-sample-management.md [hunk 3]
```diff
@@ -25,39 +25,78 @@ nav_order: 2
...
+[Photobleaching](panel-photobleaching.html), 
+[Denoising](panel-denoising.html) and 
+[Find states](panel-find-states.html) to the selected molecule.
+
 
 ---
 
 ## Current molecule
 
+Defines the molecule index of the currently selected molecule.
```
Comment (pflanzenhueter) [inline docs/trace-processing/panels/panel-sample-management.md]: Have you deleted this part later on? Its missing in the final pull-request?

File: docs/histogram-analysis/workflow.md (hunk 2 above)
Comment (mca-sh) [reply docs/histogram-analysis/workflow.md]: I did not find a better way than generating image files to display proper equations in html. For this I use the online tool at https://www.codecogs.com/latex/eqneditor.php

**Pull request merged**